packages/keycloak-user-management/src/components/CreateEditUser/index.tsx

```diff
@@ -91,7 +86,7 @@ const CreateEditUser: React.FC<CreateEditPropTypes> = (props: CreateEditPropType
    */
   React.useEffect(() => {
     if (userId && !keycloakUser) {
-      const serve = new serviceClass(accessToken, KEYCLOAK_URL_USERS, keycloakBaseURL);
+      const serve = new serviceClass(KEYCLOAK_URL_USERS, keycloakBaseURL);
```

Comment:
why remove access token?

Reply:
the service class has been refactored to no longer pass access token, see [here](https://github.com/OpenSRP/web/pull/412/files#diff-0fd8769ef96cf84dd274b1cecd582da912591e182e253a03169a59045ecab9c3R302-R304)

Reply:
then do we need to pass the access token as a prop?

Reply:
that is being handled [here](https://github.com/OpenSRP/web/pull/412/files#diff-0fd8769ef96cf84dd274b1cecd582da912591e182e253a03169a59045ecab9c3R304)

Reply:
i mean do we still need to retain the access token prop in this file.

Reply:
addressed